app.rb

```diff
@@ -0,0 +1,94 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'active_record'
+
+
+require_relative 'db/connection'
+require_relative 'models/pokemon'
+require_relative 'models/trainer'
+
+get '/pokemon' do
+	@the_list = Pokemon.all.order(:id)
+	erb :all
+end
+get '/pokemon/new'do
+	erb :add_a_pokemon
+end
+
+post '/pokemon' do 
+	@new_poke = Pokemon.create(name: params[:name],poke_type: params[:poke_type],cp: params[:cp],img_url: params[:img])
```

Comment:
How could you update your form's `name` attributes so you don't have to individually assign all these properties?